build.gradle.kts

```diff
@@ -0,0 +1,140 @@
+import com.android.build.gradle.BaseExtension
+import com.dicedmelon.gradle.jacoco.android.JacocoAndroidUnitTestReportExtension
+import com.jfrog.bintray.gradle.BintrayExtension
+import com.novoda.gradle.release.PublishExtension
+import org.jetbrains.kotlin.gradle.dsl.Coroutines
+import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+        google()
+    }
+
+    dependencies {
+        classpath("com.android.tools.build:gradle:3.1.3")
```

Comment:
Can we also move this into `dependencies.kt` too? maybe we can call `BulidScript.androidGradle`? Any specific reason you want it to hang around here? 😄

Reply:
Yes, we can. The commit 76388d3 moves them to `Plugins` object. The name can be changed if you think `BulidScript` is better. :+1: